src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecAfterPaymentCallbackHandler.java

```diff
@@ -45,14 +47,16 @@ public List<CaseEvent> handledEvents() {
     }
 
     private CallbackResponse changeStateToCaseIssued(CallbackParams callbackParams) {
-        Long caseId = callbackParams.getCaseData().getCcdCaseReference();
         CaseData caseData = callbackParams.getCaseData();
         CaseData.CaseDataBuilder dataBuilder = caseData.toBuilder();
         dataBuilder.businessProcess(BusinessProcess.ready(CREATE_CLAIM_SPEC_AFTER_PAYMENT));
-
+        if (Objects.isNull(caseData.getRespondent1ResponseDeadline())) {
+            dataBuilder.respondent1ResponseDeadline(deadlinesCalculator.plus28DaysAt4pmDeadline(
```

Comment:
I disagree with this approach. because it will cause issues in live lr. we are setting the response deadline date much later in lr. this handler is used in live and in release1

Reply:
I dont think it is an issue if this code is behind the R2 feature toggle